buildwin.bat

```diff
@@ -0,0 +1,11 @@
+
+set BORG_OPENSSL_PREFIX=D:\OpenSSL-Win64
```

Comment:
is pkg-config a thing under windows?

on linux (and others) it is used to locate headers and libraries in a standard way (and openssl supports it, as well as most other libs we use).

Reply:
There are multiple tools that try to fill that spot. They have varying usability.
It is often best to not try to use any of them.